Linkbrary/javascript/error.js

```diff
@@ -0,0 +1,49 @@
+// 에러조건
+// id 에러 상황
+function idError(event){ 
+  event.preventDefault(); // 새로고침 막기
+  if(window.location.pathname==='/Linkbrary/signin.html'){
+    if(ID_INPUT.value?.trim()==false){
+      showidErrorMessage("이메일을 입력해 주세요.");
+      return;
+    }
+    else if(!isValidEmail(ID_INPUT.value)){
+      showidErrorMessage("올바른 이메일 주소가 아닙니다.");
+    }
+  }else{
+    if(ID_INPUT.value?.trim()==false){
+      showidErrorMessage("이메일을 입력해 주세요.");
+      return;
+    }
+    else if(!isValidEmail(ID_INPUT.value)){
+      showidErrorMessage("올바른 이메일 주소가 아닙니다.");
+    }
+    else if(ID_INPUT.value === TEST_EMAIL){
+      showidErrorMessage("이미 사용 중인 이메일입니다.");
+    }
+  }
```

Comment:
코드를 보면 6~12줄이랑 14~19번째줄이랑 완전히 동일해서 if 문이 의미가 없늕 거 같습니다!

```suggestion
    if(!ID_INPUT.value?.trim()){
      showidErrorMessage("이메일을 입력해 주세요.");
      return;
    }
    
    if(!isValidEmail(ID_INPUT.value)){
      showidErrorMessage("올바른 이메일 주소가 아닙니다.");
      return;
    }
    
    if(window.location.pathname!=='/Linkbrary/signin.html' && ID_INPUT.value === TEST_EMAIL) {
       showidErrorMessage("이미 사용 중인 이메일입니다.");
      }
  }
```